.github/workflows/ci.yml

```diff
@@ -164,3 +164,26 @@ jobs:
       uses: coverallsapp/github-action@master
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
+  # Make sure the library builds with all dependencies downgraded to their
+  # oldest versions allowed by the semver spec. This ensures we have not
+  # under-specified any dependency
+  minimal-versions:
```

Comment:
This needs to be referenced on line 26 to block merge

Reply:
Ah, cool. Fixed!